cuegui/cuegui/HostMonitorTree.py

```diff
@@ -120,19 +136,19 @@ def __init__(self, parent):
                            "in every 60 seconds so a number larger than this\n"
                            "indicates a problem")
         self.addColumn("Hardware", 70, id=15,
-                       data=lambda host: str(host.data.state),
+                       data=lambda host: HOST_HARDWARE_MAP[host.data.state],
```

Comment:
Can we use `HardwareState.Name()` here instead of maintaining a separate map (and so on for the other enums)? It feels like the map could easily become out of date.

Reply:
Yes! I hated these maps, I'm glad you brought this up. Thanks, done.